build.sbt

```diff
@@ -130,30 +132,46 @@ val benchmarks = project
     skip in publish := true,
   )
 
+val integrations = file("integrations")
+
 lazy val sbtBloop10 = project
   .enablePlugins(ScriptedPlugin)
-  .in(file("integrations") / "sbt-bloop")
+  .in(integrations / "sbt-bloop")
   .settings(BuildDefaults.scriptedSettings)
   .settings(sbtPluginSettings("1.1.4", jsonConfig212))
   .dependsOn(jsonConfig212)
 
 // Let's remove scripted for 0.13, we only test 1.0
 lazy val sbtBloop013 = project
   .disablePlugins(ScriptedPlugin)
-  .in(file("integrations") / "sbt-bloop")
-  .settings(scalaVersion := Keys.scalaVersion.in(jsonConfig210).value)
+  .in(integrations / "sbt-bloop")
+  .settings(scalaVersion := scala210Version)
   .settings(sbtPluginSettings("0.13.17", jsonConfig210))
   .dependsOn(jsonConfig210)
 
-val mavenBloop = project
-  .in(file("integrations") / "maven-bloop")
+val mavenBloop210 = project
+  .in(integrations / "maven-bloop")
+  .disablePlugins(ScriptedPlugin)
+  .dependsOn(jsonConfig210)
+  .settings(
+    name := "maven-bloop",
+    scalaVersion := scala210Version,
+    target := (integrations / "target" / "maven-bloop-2.10").getAbsoluteFile
+  )
+  .settings(BuildDefaults.mavenPluginBuildSettings)
+
+val mavenBloop212 = project
```

Comment:
There is no need to keep the maven bloop plugin for 2.12 since our dependency will always be 2.10, so we can remove the redefinition of `name` in mavenBloop210 and make it `mavenBloop` instead.

Reply:
Got it